gatsby-node.js

```diff
@@ -65,11 +68,15 @@ exports.createPages = async ({ graphql, actions }) => {
 
   _.each(result.data.allContentfulSpeciality.edges, edge => {
     if (edge.node.slug && edge.node.generate) {
+      const extraTags = getTagsForTitle(edge.node.title)
+
       createPage({
         path: `/speciality/${edge.node.slug}/`,
         component: slash(specialityTemplate),
         context: {
-          id: edge.node.id
+          id: edge.node.id,
```

Comment:
by passing these (limit of results and tags), we can access them from the PageQuery level and use them to make the filtering on the graphql side